source/server/active_tcp_socket.cc

```diff
@@ -75,6 +74,29 @@ void ActiveTcpSocket::continueFilterChain(bool success) {
     bool no_error = true;
     if (iter_ == accept_filters_.end()) {
       iter_ = accept_filters_.begin();
+      if (listener_filter_max_read_bytes_ > 0) {
+        listener_filter_buffer_ = std::make_unique<Network::ListenerFilterBufferImpl>(
+            socket_->ioHandle(), listener_.dispatcher(),
+            [this](bool error) {
+              socket_->ioHandle().close();
+              error ? listener_.stats_.downstream_peek_error_.inc()
```

Comment:
For actions with side-effects and no value, I think this should be `if/else` not `? :`.

Reply:
got it